hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2956,16 +2956,16 @@ public String getRatisRoles() {
         leaderId = omRatisServer.getLeader();
         if (leaderId == null) {
           LOG.error("No leader found");
-          return "Exception: Not a leader";
+          return Collections.emptyList();
```

Comment:
How to show "Not a leader" or any exception occured in system? if exeption is happening, instead of empty list, need throw exception.